.serena/memories/skills-documentation.md

```diff
@@ -297,6 +323,27 @@ Implement user authentication:
 3. [Depends: 2] Build login UI components
 4. [Depends: 2, 3] Integration test auth flow
 5. [Depends: 4] Deploy to staging
+
+# GOOD: Workflow with rate limit awareness
+on:
+  schedule:
+    - cron: '0 */4 * * *'  # Every 4 hours, not every push
+  workflow_dispatch:       # Manual trigger available
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    # Concurrency prevents overlapping runs
+    concurrency:
+      group: pr-maintenance
+      cancel-in-progress: false
+    steps:
+      - name: Check rate limit before proceeding
+        run: |
+          REMAINING=$(gh api rate_limit --jq '.resources.core.remaining')
```

Comment:
@rjmurillo-bot Note that this will need an update after #273 and #275 land in main

Reply:
Acknowledged. This section will be updated after PR #273 (PowerShell compatibility improvements) and PR #275 (test infrastructure updates) are merged.

Tracking this for future update.